webapp/models.py

```diff
@@ -26,7 +28,19 @@ class Asset(Base):
     __tablename__ = "asset"
 
     id = Column(Integer, primary_key=True)
-    created = Column(DateTime, nullable=False)
+    author = Column(String, nullable=True)
+    title = Column(String, nullable=True)
+    asset_type = Column(String, nullable=True)
+    product = Column(String, nullable=True)
+    portfolio = Column(String, nullable=True)
+    copydoc = Column(String, nullable=True)
```

Comment:
from the design I couldn't find a "Copy Doc" field. Did you mean "Google Drive link"?